core/src/services/enrichments/vulnerabilities/epss/client.rs

```diff
@@ -0,0 +1,85 @@
+use crate::Error;
+use platform::hyper::ContentType;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug)]
+pub(in crate::services::enrichments::vulnerabilities::epss) struct Client {
+    inner: platform::hyper::Client,
+}
+
+impl Client {
```

Comment:
I like the naming convention of "Client" and using the crate/module to assist with resolution.  It also lets developers using the struct to name the client anything they want in th `use` clause.